src/Maestro/maestro-angular/src/app/page/build/build.component.ts

```diff
@@ -271,17 +294,17 @@ export class BuildComponent implements OnInit, OnChanges {
     if (!build.azureDevOpsBranch) {
       throw new Error("azureDevOpsBranch undefined");
     }
-    return this.buildStatusService.getBranchStatus(build.azureDevOpsAccount, build.azureDevOpsProject, build.azureDevOpsBuildDefinitionId, build.azureDevOpsBranch, 5)
+    return this.buildStatusService.getBranchStatus(build.azureDevOpsAccount, build.azureDevOpsProject, build.azureDevOpsBuildDefinitionId, build.azureDevOpsBranch, 5, "completed")
```

Comment:
How do we know that the 5 returned builds are the ones we are interested in when there's no ordering specified anywhere? And what if there is more than 5 completed builds that are newer than `build`? Won't we miss some useful information here?

To be clear - I don't know what are the business requirements here so maybe it's good enough.

Reply:
I am also curious about logic here. But fixing this was not goal of this task so I'd rather left it as it.
I think, not 100% sure, that in vast majority in cases it will work because:
-  it looks for newest failed build from 5 completed
- there would have to be more than 5 successfully completed builds since this build
- but after build is successful Arcade SDK reports it into maestro and that build is the ref build towards which are failed builds looked up